src/main/java/com/minecolonies/coremod/network/messages/ChangeFreeToInteractBlockMessage.java

```diff
@@ -126,22 +126,22 @@ public void messageOnServerThread(final ChangeFreeToInteractBlockMessage message
             {
                 if (!(message.pos.getX() == 0 && message.pos.getZ() == 0 && message.pos.getY() == 0))
                 {
-                    colony.addFreePosition(message.pos);
+                    colony.addFreeBlock(message.block);
                 }
                 else
                 {
-                    colony.addFreeBlock(message.block);
+                    colony.addFreePosition(message.pos);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this line start at column 11. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)